templates/prescription/general_edit.html

```diff
@@ -342,7 +342,7 @@
     // for text boxes
     if ($('#'+objID).is('input')) {
         if ($('#'+objID).val() == "") {
-            alert({/literal}{xl t='Missing a required field and will be highlighted'|js_escape}{literal});
+            alert({/literal}{'Missing a required field and will be highlighted'|xlt|js_escape}{literal});
```

Comment:
This will over escape. If we need to do this, will instead need to be:
```
alert({/literal}{'Missing a required field and will be highlighted'|xlj}{literal});
```

Reply:
Brady, try changing from single quotes to double.

Reply:
didn't work,
This is interesting; it appears that the smarty modifier may actually work on the parameter for the main function(that would just be bizarre). Still trying to figure out following explanation:
https://www.smarty.net/docsv2/en/language.modifiers.tpl

Reply:
json_encode should not work well with single quoted strings. May not be this issue but should bare in mind.

Reply:
Another helpful linkL
https://www.smarty.net/docsv2/en/language.combining.modifiers.tpl

It looks like the intent is to use the modifier on the t parameter rather than the return of the xl function, which is just strange. And haven't found a way to not do this. May need to make smarty xla/xlt/xlj functions to use in place of the xl (not a big deal to do, but just odd that we need to do this).

Reply:
And then always be on the lookout in smarty when use a modifier on a function that has a parameter in it (sigh...)

Reply:
ok, think I have this worked out in this fix:
https://github.com/openemr/openemr/pull/2160
(and will plan to change all other places where using a modifier on the xl smarty function calls)